src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -4307,7 +4324,9 @@ int32_t NaClSysExecve(struct NaClAppThread *natp, void *pathname, void *argv, vo
 }
 
 int32_t NaClSysExecv(struct NaClAppThread *natp, void *pathname, void *argv) {
```

Comment:
You might also want to change the ```void *```s in this signature for consistency and clarity.